### ISSUE TITLE
Question about reproducing the reported results

### ISSUE THREAD

**Issue by siyuch-fdu** [aniri15/OPD-V#1]
Dear Authors,

Thank you for this excellent work!

We are having some difficulty reproducing the results reported in your paper. After training on the 6.2K dataset, our accuracy on all six test sets falls below the numbers reported in the paper as shown below (Line3). We used your released code without any hyperparameter tuning.

Could you share any guidance on how to reproduce the reported accuracy? We would be happy to provide our full training logs and configuration if that would help.

Thank you very much for your time, and we look forward to your reply.

Best regards!

<img width="1232" height="136" alt="Image" src="https://github.com/user-attachments/assets/fe2a80e8-75c4-45cf-a0d6-d1c958649c7b" />
